### PR TITLE
Fix pylint warning from master branch

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**
## Description

Pin pylint version for the CI, and fix the current warning in the codebase.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :bug: Bug fix          |
| ✓   | :hammer: Refactoring   |

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -109,6 +109,7 @@ jobs:
       - name: Run formatting check
         run: |
           . venv/bin/activate
+          pip install -r requirements_test_pre_commit.txt # Pylint is a system hook we need it
```
Comment (cdce8p) [inline .github/workflows/ci.yaml:112]: I haven't looked at the action logs, but we do restore the Python venv here which includes `pylint` already.

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -334,7 +335,8 @@ jobs:
         run: >-
           echo "::set-output name=key::venv-${{ env.CACHE_VERSION }}-${{
             hashFiles('setup.cfg', 'requirements_test_min.txt',
-          'requirements_test_brain.txt', 'requirements_test_pre_commit.txt') }}"
+           'requirements_test_brain.txt')
+          }}"
```
Comment (cdce8p) [inline .github/workflows/ci.yaml:339]: Previously, we didn't run the extra tests for `pypy` which is quite reasonable I believe.

**Review by cdce8p — CHANGES_REQUESTED** [review]
Would you mind adding the "fix" for travis as well? https://github.com/PyCQA/astroid/commit/f157130d665ffb271e426964dc9aa105b395cb73

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (Pierre-Sassoulas) [reply .github/workflows/ci.yaml]: I removed that

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (Pierre-Sassoulas) [reply .github/workflows/ci.yaml]: Also removed

**Comment by Pierre-Sassoulas** [comment]
> Would you mind adding the "fix" for travis as well? [f157130](https://github.com/PyCQA/astroid/commit/f157130d665ffb271e426964dc9aa105b395cb73)

I pushed it directly on master.

**Comment by Pierre-Sassoulas** [comment]
I think we should also do https://github.com/PyCQA/pylint/issues/4390

**Review by cdce8p — APPROVED** [review]
👍🏻

**Pull request merged**